### ISSUE TITLE
unrecognized unsigned message sub_message

### ISSUE THREAD

**Issue by bogdanripa** [Teslemetry/node-vehicle-command#1]
When calling "sendMessage" I get the following error. Any idea why?

{"response":null,"error":"unrecognized unsigned message sub_message","error_description":"","txid":"ce33beccb5171fd80b172a0aa24375f9"}

**Comment by Bre77** [comment]
Yes, because I never was able to get this implementation to work.

It's on my to-do list to revisit it eventually.

**Comment by bogdanripa** [comment]
I guess you stopped working in this? :)

**Comment by Bre77** [comment]
> I guess you stopped working in this? :)

I stopped working on this library, however I did figure it out.

This is the python implementation which I have published: https://github.com/Teslemetry/python-tesla-fleet-api

I then reimplemented it in TypeScript for my Fastify API, but that's not open source.
